modules/social_features/social_album/social_album.install

```diff
@@ -64,28 +63,21 @@ function social_album_install() {
  * @param array $paths
  *   List of blocks which should be updated and paths which should be added.
  *
+ * @deprecated in social:11.4.0 and is removed from social:12.0.0. Use
+ *   _social_core_blocks_paths instead.
+ *
```

Comment:
@ksis Please add `@see https://github.com/drupal-graphql/graphql/pull/2985` to fix PHPCS check.